tests/test_main.py

```diff
@@ -777,9 +777,9 @@ class Model(BaseModel):
         model_config = ConfigDict(use_enum_values=True)
 
     m = Model(baz=FooEnum.foo)
-    assert m.model_dump() == {'baz': FooEnum.foo, 'boo': 'hoo'}
+    assert m.model_dump() == {'baz': 'foo_value', 'boo': 'hoo'}
     assert m.model_dump(mode='json') == {'baz': 'foo_value', 'boo': 'hoo'}
-    assert m.baz.value == 'foo_value'
+    assert m.baz == 'foo_value'
```

Comment:
Hmm, this existing test makes me think that this would technically be a breaking change...

Reply:
Looks like this test was initially introduced in https://github.com/pydantic/pydantic/pull/1838/files, where I think this assertion:

```py
assert m.baz.value == 'foo_value'
```

Was definitely a mistake

Reply:
So I'll modify my claim - I think this is definitely a fix, not a breaking change.